### ISSUE TITLE
Cuboid>translate.py unused variables and imports

### ISSUE THREAD

**Issue by repositony** [GEOUNED-code/GEOUNED#33]
Couple of unused code in this file, may want to remove them if it was intended:

- `dist1` and `dist2` are never used.

```python
def isInverted(solid):
    face = solid.Faces[0]
    Range = face.ParameterRange
    u = (Range[1] + Range[0]) / 2.0
    v = (Range[3] + Range[2]) / 2.0

    dist1 = face.CenterOfMass.distanceToPoint(solid.BoundBox.Center)
    dist2 = face.CenterOfMass.add(face.normalAt(u, v).multiply(1.0e-6)).distanceToPoint(
        solid.BoundBox.Center
    )
    point2 = face.CenterOfMass.add(face.normalAt(u, v).multiply(1.0e-6))

    if solid.isInside(point2, 1e-7, False):
        return True
    else:
        return False
```

- half of these imports are never used (`isInLine`, `signPlane`, `isSameValue`)

```python
from GEOUNED.Utils.BasicFunctions_part1 import (
    isParallel,
    isOposite,
    isInLine,
    signPlane,
    isSameValue,
)
```

**Comment by repositony** [comment]
See PR #35

**Comment by repositony** [comment]
Merged in commit 4db7ce255b1c1fd300ae1e5f5b68ba28ade5674d
